### ISSUE TITLE
Weird stepOut behavior on last method in interface (alphabetically) in go1.13

### ISSUE THREAD

**Issue by nd** [go-delve/delve#2079]
1. What version of Delve are you using (`dlv version`)?

```
Delve Debugger
Version: 1.4.1
Build: 708eadd5538edd222d8553b08b47c295474f38fa
```

2. What version of Go are you using? (`go version`)?

```
go version go1.13.12 windows/amd64
```

3. What operating system and processor architecture are you using?

windows/amd64

4. What did you do?

Debug the program:

```
package main

type I interface {
	foo()
	bar()
}

type T struct {}

func (t T) foo() {
	println("ok")
}

func (t T) bar() {
	println("ok")
}

func main() {
	t := T{}
	var i I = t
	i.bar() //break
 	i.foo()
}
```

Once breakpoint is reached, step into method bar, and then stepOut. Current location is reported to be main.main as expected. Step into method foo() and stepOut.

5. What did you expect to see?

Current location is main.main.

6. What did you see instead?

Current location is main.(T*).foo in main.go:1.

Interestingly if I add a method which is alphabetically after foo (e.g. foo1 or fop) to the interface and implement it for T, then stepOut from foo starts to work properly. Adding a method which is alphabetically before foo() doesn't change foo's stepOut behavior. Also stepOut seems to work fine in go1.14.

Full log: https://gist.github.com/nd/04a2e4685c496b995c6c44e43dbda853

**Comment by chainhelen** [comment]
I check it on linux,  same issue when go1.13.12 linux/amd64 && dlv 1.4.1

**Comment by aarzilli** [comment]
I think that our interpretation of debug_line matches the specification. Seeing that:

1. this has been fixed in 1.14 because the code that emits debug_line changed, and
2. the effect is just cosmetic (the autogen filter not working doesn't impact functionality)

I'd say that we shouldn't do anything about this. If you have objections please reopen.
